17-react-native/classworks/Gavruk_Alexander/screens/Details/Details.screen.js

```diff
@@ -0,0 +1,84 @@
+import React, { useCallback, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  View,
+  SafeAreaView,
+  ScrollView,
+  Image,
+  Text,
+} from 'react-native';
+import { useRoute } from '@react-navigation/native';
+import { FlatList } from 'react-native-gesture-handler';
+
+import fetchMarvel from '../../fetchMarvel';
+import { Heart } from '../../components/Heart';
+
+import * as characterActions from '../../resources/characters/characters.actions';
+import * as characterSelectors from '../../resources/characters/characters.selectors';
+import styles from './Details.styles';
+
+function DetailsScreen() {
+  const { params } = useRoute();
+  const { id, name, img, description } = params
+  const dispatch = useDispatch();
+
+  const comics = useSelector(characterSelectors.getComics);
+
+  const fetchData = useCallback(async () => {
+    dispatch(characterActions.fetchComics(fetchMarvel, `/characters/${id}/comics`));
```

Comment:
why import `fetchMarvel` in component and pass as param? we can import this fn in actions file